### ISSUE TITLE
Enable tests in auctionCancel_test.go on macox 

### ISSUE THREAD

**Issue by algonautshant** [algorand/go-algorand#2205]
## Summary

This test is disabled on Macos.
It seems to be passing and should be enabled. 

TestStartAndCancelAuctionOneUserTenBids 2min 17sec

## Scope/Requirements

Further test to make sure it is not failing, and enable it.

## Urgency/Relative Priority

This was disabled for some time and only on macos. Not urgent.

**Comment by tsachizehub** [comment]
This should not fixed. Instead, we need to copy these tests along with the rest of the auction code to a new repository for archiving and remove it completely from the go-algorand.

**Comment by winder** [comment]
@tsachiherman could you elaborate? Is the intent to simply extract the code and archive it somewhere or should it also be functioning?

**Comment by tsachizehub** [comment]
Just archive it. At this point, we don't want to spend engineering time on fixing code that is less than likely to be used. We still want to have this, in case someone would wonder "how did we conduct the original auction?".

If we will need it, we can always spend the time later on.

**Comment by winder** [comment]
@tsachiherman is this repo sufficient? It has the added benefit of sorta working: https://github.com/algorand/auction-tools

**Comment by tsachizehub** [comment]
yes, it would be good. However, we don't want to restructure anything. We want to move the files with the existing directory structure ( including all the ./script ) files, so that we would have the option to restore these easily.

This *should not* be a project, but rather a half a day of work moving the files. No need to work in having these being compilable on the target repository.

**Comment by winder** [comment]
Perfect:
https://github.com/algorand/auction-tools/pull/2
https://github.com/algorand/go-algorand/pull/2261

**Comment by algonautshant** [comment]
This test is not failing (20 consecutive runs passed) using  8590327808
2.6.0.dev [HEAD] (commit #8fe22d4e)

**Comment by yaovi-a** [comment]
Auction test has been pulled out -- no longer needed.
